### ISSUE TITLE
我想问一下在iou_loss下的问题

### ISSUE THREAD

**Issue by guotongjian** [PaddlePaddle/PaddleDetection#2463]
iou_loss中，有这么一段
```
class IouLoss(object):
    """
    iou loss, see https://arxiv.org/abs/1908.03851
    loss = 1.0 - iou * iou
    Args:
        loss_weight (float): iou loss weight, default is 2.5
        max_height (int): max height of input to support random shape input
        max_width (int): max width of input to support random shape input
        ciou_term (bool): whether to add ciou_term
        loss_square (bool): whether to square the iou term
    """
```
我想问一下：
1、loss计算公式为什么是1-iou的平方项，这样的原理是什么，所给论文好像没有提到这点，不是1-iou吗？
2、ciou_term是指可以将普通iouloss变成ciou loss，这样可以增加一个ciou loss分支（除xywh，置信度、分类损失），是这个意思吗，我在yolov3中将loss_square设置为false，ciou设置为true，好像效果没有原始的效果好

**Comment by liuhuiCNN** [comment]
（1）两个都是关于iou的减函数（loss关于iou的函数，需要满足iou越大-->loss越小 ），本质上差别不大。采用1-iou的平方项，是连续函数，梯度是平滑的，避免1-iou的不连续性。
（2）是的，这样相等于多增加一个loss，建议将loss_square设置为true试试。在具体数据上，此外loss中的超参数也可以调节一下试试。
